model/src/main/scala/explore/model/ProposalDetails.scala

```diff
@@ -4,21 +4,41 @@
 package explore.model
 
 import cats._
+import coulomb._
+import coulomb.accepted.Percent
+import coulomb.time.Hour
+import eu.timepit.refined.api.Refined
+import eu.timepit.refined.numeric.Interval
+import eu.timepit.refined.types.numeric.NonNegDouble
 import explore.model.enum._
+import lucuma.core.model.Partner
 import lucuma.core.model.StandardUser
 import monocle.macros.Lenses
 
 @Lenses
 final case class ProposalDetails(
-  title:         String,
-  pi:            StandardUser,
-  proposalClass: ProposalClass,
-  category:      TacCategory,
-  toOActivation: ToOActivation,
-  keywords:      Set[Keyword],
-  abstrakt:      String
+  title:          String,
+  pi:             StandardUser,
+  proposalClass:  ProposalClass,
+  category:       TacCategory,
+  toOActivation:  ToOActivation,
+  keywords:       Set[Keyword],
+  abstrakt:       String,
+  partnerSplits:  List[PartnerSplit],
+  band1And2Hours: ProposalDetails.NonNegHour,
+  band3Hours:     ProposalDetails.NonNegHour
 )
 
 object ProposalDetails {
+  type NonNegHour = Quantity[NonNegDouble, Hour]
```

Comment:
If/when this data is added to lucuma-common, we'll need to find a home for this type and the ones in PartnerSplit. I'm not sure where the best place is until then.